doc/src/docs/InputOutputReference/01a-InputOutputReference.md

```diff
@@ -20045,10 +20051,9 @@ SurfaceContaminantSourceAndSink:GenericContaminant:BoudaryLayerDiffusion,
     2.0;                    !- Henry adsorption constant or partition coefficient
 ```
 
+### SurfaceContaminantSourceAndSink:Generic:BoundaryLayerDiffusion Outputs
 
-### SurfaceContaminantSourceAndSink:GenericContaminant:BoundaryLayerDiffusion Outputs
-
-When a SurfaceContaminantSourceAndSink:GenericContaminant:BoudaryLayerDiffusion object is specified, the following output variables are available:
+When a SurfaceContaminantSourceAndSink:Generic:BoudaryLayerDiffusion object is specified, the following output variables are available:
```

Comment:
@mjwitte Done. Fortunately, the word in idd is correct. No idd changes are needed.